docs/admin/install/slack_setup.md

```diff
@@ -28,24 +30,221 @@ PLUGINS_CONFIG = {
 }
 ```
 
+!!! info
+    When running Slack in Socket Mode, Nautobot ChatOps Plugin can specify where the static files are hosted.
+    This is helpful for giving Slack access to only the server hosting Nautobot Static Files. If Django Storage
+    is configured, Nautobot ChatOps Plugin will use the static host from Django Storage. This is optional as
+    ChatOps only uses static files for sending logos with messages. To configure the static files host, use
+    `slack_socket_static_host` ex. `slack_socket_static_host: 'https://example.com/static/'`.
```

Comment:
This is still a bit confusing to me. If you're *not* using `django-storage`, is `slack_socket_static_host` required to get logos to work? Or under what circumstances should/shouldn't you need to configure this setting?

Reply:
Yeah, so if you want the logos to work, but aren't using a publicly facing django-storage, then you will need to provide a publicly accessible host. I will try to rephrase this.